devnet/node.yaml

```diff
@@ -68,11 +71,13 @@ spec:
               name: node-rundir
             - mountPath: /tmp/mounted-keys/wormchain
               name: node-wormchain-key
+            - mountPath: /app/node/config
+              name: node-config
           command:
             - /guardiand
             - node
-            - --ethRPC
-            - ws://eth-devnet:8545
+            # - --ethRPC
```

Comment:
Why did this get commented out?

Reply:
Pretty sure because it is already specified in the config file here:

https://github.com/wormhole-foundation/wormhole/pull/3710/files#diff-9ca9c1f4adddc2b4c6064f90ce44be7c99f78d4b4586c8349d62f873e8637a3eR1